<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MashWaterDialog</class>
 <widget class="QDialog" name="MashWaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mash Water Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="strikeWaterGroupBox">
     <property name="title">
      <string>Strike &amp;Water</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="strikeWaterVolumeLabel">
        <property name="text">
         <string>Quarts of &amp;water</string>
        </property>
        <property name="buddy">
         <cstring>strikeWaterVolumeDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="strikeWaterVolumeDoubleSpinBox">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>80.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.250000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="grainMassLabel">
        <property name="text">
         <string>Pounds of &amp;grain</string>
        </property>
        <property name="buddy">
         <cstring>grainMassDoubleSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="grainMassDoubleSpinBox">
        <property name="minimum">
         <double>0.250000000000000</double>
        </property>
        <property name="maximum">
         <double>60.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="targetTempLabel">
        <property name="text">
         <string>&amp;Target temp</string>
        </property>
        <property name="buddy">
         <cstring>targetStrikeTempSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="targetStrikeTempSpinBox">
          <property name="minimum">
           <number>80</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>153</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="targetStrikeTempSlider">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>80</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>153</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="grainTempLabel">
        <property name="text">
         <string>G&amp;rain temp</string>
        </property>
        <property name="buddy">
         <cstring>grainTempSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="grainTempSpinBox">
          <property name="minimum">
           <number>32</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="grainTempSlider">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>30</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ratioLabelLabel">
        <property name="text">
         <string>Water/grain ratio:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="ratioLabel">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>2.0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="strikeWaterTempLabelLabel">
        <property name="text">
         <string>Strike water temperature:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="strikeWaterTempLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>161.30</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="secondStepGroupBox">
     <property name="title">
      <string>&amp;Second Step</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="step2TargetTempLabel">
        <property name="text">
         <string>Step &amp;2 target temp</string>
        </property>
        <property name="buddy">
         <cstring>step2TargetTempSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="step2TargetTempSpinBox">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>140</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="step2TargetTempSlider">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>140</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="step2WaterTempLabel">
        <property name="text">
         <string>I&amp;nfusion water temp</string>
        </property>
        <property name="buddy">
         <cstring>step2WaterTempSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSpinBox" name="step2WaterTempSpinBox">
          <property name="minimum">
           <number>140</number>
          </property>
          <property name="maximum">
           <number>212</number>
          </property>
          <property name="value">
           <number>210</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="step2WaterTempSlider">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>140</number>
          </property>
          <property name="maximum">
           <number>212</number>
          </property>
          <property name="value">
           <number>210</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="step2WaterVolumeLabel">
        <property name="text">
         <string>6.17</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="step2WaterVolumeLabeLabel">
        <property name="text">
         <string>Volume of water to add:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="thirdStepGroupBox">
     <property name="title">
      <string>&amp;Third Step</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="step3TargetTempLabel">
        <property name="text">
         <string>Step &amp;3 target temp</string>
        </property>
        <property name="buddy">
         <cstring>step3TargetTempSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSpinBox" name="step3TargetTempSpinBox">
          <property name="minimum">
           <number>120</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>158</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="step3TargetTempSlider">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>120</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>158</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="step3WaterTempLabel">
        <property name="text">
         <string>Infu&amp;sion water temp</string>
        </property>
        <property name="buddy">
         <cstring>step3WaterTempSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSpinBox" name="step3WaterTempSpinBox">
          <property name="minimum">
           <number>140</number>
          </property>
          <property name="maximum">
           <number>212</number>
          </property>
          <property name="value">
           <number>210</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="step3WaterTempSlider">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>140</number>
          </property>
          <property name="maximum">
           <number>212</number>
          </property>
          <property name="value">
           <number>210</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="step3WaterVolumeLabelLabel">
        <property name="text">
         <string>Volume of water to add:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="step3WaterVolumeLabel">
        <property name="text">
         <string>6.29</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="moreButton">
       <property name="text">
        <string>&amp;More &gt;&gt;</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MashWaterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MashWaterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>secondStepGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>thirdStepGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
